vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/ChartVariant.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
Please update the license header to CVDL (commercial license) as Charts is a Pro component. 
You can copy a corresponding comment from one of the other files in `vaadin-charts-flow`